### PR TITLE
add a decibel conversion command

### LINKED ISSUE #231: New command: dB conversions
specificially requested: dBm <-> W or mW

Could also do: dBW, dBd, dBi, dBA, etc

### PR DESCRIPTION
**Pull request by classabbyamp**
fixes #231

### CODE REVIEW

**Comment by classabbyamp** [comment]
note to self: update changelog

File: exts/conv.py [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+
+import common as cmn
+from utils import dbconv
+
+
+class ConvCog(commands.Cog):
+    def __init__(self, bot: commands.Bot):
+        self.bot = bot
+
+    @commands.command(name="dbconv", aliases=["db"], category=cmn.cat.ref)
```
Comment (classabbyamp) [inline exts/conv.py:21]: change alias to dbc?

File: exts/conv.py [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+    async def _db_conv(self, ctx: commands.Context, value: float, unit1: str, unit2: str):
+        """
+        Convert between decibels and scalar values for voltage, power, and antenna gain.
```
Comment (classabbyamp) [inline exts/conv.py:24]: help text maybe should show a reference of dB values (what they're relative to)

File: utils/dbconv.py [hunk 3]
```diff
@@ -0,0 +1,139 @@
...
+            # V/W to dB
+            elif not unit1.is_db and unit2.is_db:
+                if unit1.type == UnitType.voltage:
+                    return _calc_volt_db(initial * unit1.mult, unit2.mult)
+                elif unit1.type == UnitType.power:
+                    return _calc_power_db(initial * unit1.mult, unit2.mult)
+        raise ValueError(f"Can't convert between {unit1} and {unit2}")
+
+
+class Unit:
```
Comment (classabbyamp) [inline utils/dbconv.py:57]: have subclasses for voltage, power, antenna?

File: utils/dbconv.py [hunk 4]
```diff
@@ -0,0 +1,139 @@
...
+
+
+class DbConverter:
+    def __init__(self, value: float, unit1: str, unit2: str):
+        self.initial: float = value
+        self.unit1: Unit = Unit(unit1)
+        self.unit2: Unit = Unit(unit2)
+        self.converted: float = self._convert(self.initial, self.unit1, self.unit2)
+
+    def _convert(self, initial: float, unit1, unit2):
```
Comment (classabbyamp) [inline utils/dbconv.py:25]: change `DbConverter` to a function

File: utils/dbconv.py (hunk 3 above)
Comment (classabbyamp) [reply utils/dbconv.py:57]: maybe make this a converter, too

**Review by classabbyamp — COMMENTED** [review]
comments from @0x5c 

move to `exts/conv.py`, make it `dbconv.py`

**Review by 0x5c — APPROVED** [review]

**Pull request merged**